contracts/src/v0.8/dev/CanaryUpkeep.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+import "../interfaces/KeeperCompatibleInterface.sol";
+import "../interfaces/KeeperRegistryInterface.sol";
+import "../ConfirmedOwner.sol";
+
+/**
+ * @notice A canary upkeep which requires a different keeper to service its upkeep at an interval. This makes sure that
+ * all keepers are in a healthy state.
+ */
+contract CanaryUpkeep is KeeperCompatibleInterface, ConfirmedOwner {
+  uint256 private s_keeperIndex;
+  uint256 private s_interval = 300;
+  uint256 private s_timestamp;
+  KeeperRegistryInterface private immutable s_keeperRegistry;
```

Comment:
immutable variables should be named as i_
[Reference solidity guidelines](https://www.notion.so/chainlink/Solidity-Style-Guide-daa6418c56c04a45a2ff8c2655824c86#8cb556378f5041948cc3856e75fce012)

Reply:
Done